src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -85,6 +86,12 @@ def get_default_detection_settings():
                 "allowed_span_ops": ["ui.long-task", "ui.sentry.long-task"],
             }
         ],
+        DetectorType.RENDER_BLOCKING_ASSET_SPAN: {
+            "fcp_minimum_threshold": 2000.0,  # ms
+            "fcp_maximum_threshold": 10000.0,  # ms
+            "fcp_ratio_threshold": 0.25,
+            "allowed_span_ops": ["resource.link", "resource.script"],
+        },
```

Comment:
Making this an array like the others is awkward: the settings aren't op-specific, but even a single-item array with `settings_for_span` can't work because the FCP min/max thresholds aren't used in the context of an individual span at all. Works fine as a hash read directly, but it's inconsistent with the rest of the settings.

Reply:
Yeah we can address settings after all the other detectors are done